{"uuid": "0b0b53af-816a-4327-82c7-179dfddf1f16", "children": ["ef3ff8d1-e34d-4b51-a96b-1cae4e04c05d"], "befores": [{"name": "classSetup", "status": "broken", "statusDetails": {"message": "AttributeError: 'TestLogin' object has no attribute 'driver'\n", "trace": "  File \"C:\\Users\\ADMIN\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"C:\\Users\\ADMIN\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\ADMIN\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\ADMIN\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1125, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\ADMIN\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\fixtures.py\", line 887, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"F:\\AGEST\\Selenium\\SeleniumPython\\tests\\test_login.py\", line 16, in classSetup\n    self.login_page = LoginPage(self.driver)\n                                ^^^^^^^^^^^\n"}, "start": 1721026728030, "stop": 1721026728030}], "start": 1721026728030, "stop": 1721026728046}